src/encode.rs

```diff
@@ -154,6 +154,7 @@ impl_upstream!(u8);
 impl_upstream!(u32);
 impl_upstream!(u64);
 impl_upstream!([u8; 32]);
+impl_upstream!(Box<[u8]>);
```

Comment:
I don't know if all of the `impl_upstream!` are required for `bitcoin::` types anymore.

Reply:
Heh, good find, I removed all the ones that the compiler would let me.